docs/design/design.md

```diff
@@ -13,21 +13,21 @@
 
 - **SuperNode**: SuperNode is a long-running process with two primary responsibilities:
   - It's the tracker and scheduler in the P2P network that choose appropriate downloading net-path for each peer.
-  - It's also a CDN server that caches downloaded data from source to avoid downloading same files from source repeatedly.
+  - It's also a CDN server that caches downloaded data from source to avoid downloading the same files from source repeatedly.
 - **Dfget**: Dfget is the client of Dragonfly used for downloading files. It's similar to using wget. At the same time, it also plays the role of peer, which can transfer data between each other in p2p network.
-- **Dfdaemon**: Dfdaemon establishes a proxy between container engine and registry, as a local long-running process. It filters out layer fetching requests from all requests send by container engine when pulling images, then it uses Dfget to downloading these layers.
+- **Dfdaemon**: Dfdaemon establishes a proxy between container engine and registry, as a local long-running process. It filters out layer fetching requests from all requests send by container engine when pulling images, then it uses Dfget to download these layers.
 - **P2P**: Peer-to-Peer, a distributed application architecture, more info refers to [wiki](https://en.wikipedia.org/wiki/Peer-to-peer).
 - **Task**: A Task will store some meta info about the taskFile, pieces and something else.A Task has a one-to-one correspondence with a file on the disk which is identified by taskID. If you want to download a file from Supernode, you should register a task which means that tell the server about the file info you want to download before actually do it.
 - **DfgetTask**: A DfgetTask represents a download process initiated by Dfget or other clients. When Dfget try to download a file from p2p network, Supernode will create a DfgetTask object for managing the lifecycle of download process.
-- **Peer**: In a P2P network, peers are both suppliers and consumers of resources. So before Dfget downloads a task from Supernode, Dfget will launch a webserver that provides downloaded files for other peers to download in the P2P network and send a `Peer/Register` request to Supernode to join the P2P network. Only after that can Dfget download files from P2P network.
-- **Piece**: A piece is a part of the file that is going to download, which can be interpreted as a block of files. In dragonfly, the downloading file is not transmitted in a whole file, but in pieces.
+- **Peer**: In a P2P network, peers are both suppliers and consumers of resources. So before Dfget begains a download task from Supernode, Dfget will launch a webserver that provides downloaded files  service for other peers to download in the P2P network and send a `Peer/Register` request to Supernode to join the P2P network. Only after that can Dfget download files from P2P network.
```

Comment:
s/begains/begins?
And remove the one of the duplicated whitespaces before `service`. Thanks.